client/Stores/Query.elm

```diff
@@ -15,3 +16,13 @@ queryDecoder =
     succeed Query
         |> required "id" string
         |> required "sql" string
+        |> required "envelope" bool
+
+
+boolToString : Bool -> String
```

Comment:
Again, better move it here https://github.com/zalando-nakadi/nakadi-ui/blob/master/client/Helpers/String.elm#L182 and remove double negative `not`.